source/material_model/table.cc

```diff
@@ -280,8 +280,8 @@ namespace aspect
 // TODO: clamping into the valid range in all cases okay?
         const double pressure = std::max(min_p, std::min(p, max_p-delta_p));
 
-        Assert (pressure >= min_p, ExcMessage ("Not in range"));
-        Assert (pressure <= max_p, ExcMessage ("Not in range"));
+        Assert (pressure >= min_p, ExcMessage ("Pressure not greater than minimum value for pressure"));
+        Assert (pressure <= max_p, ExcMessage ("Pressure not less than maximum value for pressure"));
```

Comment:
Here and below, can you make it into complete sentences with a verb and a period at the end?